setup.py

```diff
@@ -17,6 +17,18 @@
         # "crytic-compile>=0.2.2",
         "crytic-compile",
     ],
+    extras_require={
+        "dev": [
+            "black>=22.3.0",
```

Comment:
We need to use a fixed version for black/pylint, otherwise we might have conflicts